tools/buildsys/src/args.rs

```diff
@@ -15,10 +15,11 @@ use url::Url;
 /// variable changes. The build type is represented with bit flags so that we can easily list
 /// multiple build types for a single variable. See `[BuildType]` and `[rerun_for_envs]` below to
 /// see how this list is used.
-const REBUILD_VARS: [(&str, u8); 15] = [
+const REBUILD_VARS: [(&str, u8); 16] = [
     ("BUILDSYS_ARCH", PACKAGE | KIT | VARIANT),
     ("BUILDSYS_CACERTS_BUNDLE_OVERRIDE", VARIANT),
     ("BUILDSYS_KITS_DIR", KIT),
+    ("BUILDSYS_EXTERNAL_KITS_DIR", KIT | VARIANT),
```

Comment:
I can't leave a suggestion for some reason, but Packages can also depend on an external kit, so I think this one is PACKAGE | KIT | VARIANT